sdk/core/azure-core-management/src/main/java/com/azure/core/management/SystemData.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.management;
+
+import java.time.OffsetDateTime;
+
+/** Metadata pertaining to creation and last modification of the resource. */
+public final class SystemData {
+
+    private String createdBy;
+
+    private CreatedByType createdByType;
+
+    private OffsetDateTime createdAt;
+
+    private String lastModifiedBy;
+
+    private CreatedByType lastModifiedByType;
+
+    private OffsetDateTime lastModifiedAt;
+
+    /**
+     * Get the identity that created the resource.
+     *
+     * @return the identity that created the resource.
+     */
+    public String createdBy() {
+        return this.createdBy;
+    }
+
+    /**
+     * Get the type of identity that created the resource.
+     *
+     * @return the type of identity that created the resource.
+     */
+    public CreatedByType createdByType() {
+        return this.createdByType;
+    }
+
+    /**
+     * Get the timestamp of resource creation (UTC).
+     *
+     * @return the timestamp of resource creation (UTC).
+     */
+    public OffsetDateTime createdAt() {
+        return this.createdAt;
+    }
+
+    /**
+     * Get the identity that last modified the resource.
+     *
+     * @return the identity that last modified the resource.
+     */
+    public String lastModifiedBy() {
+        return this.lastModifiedBy;
+    }
+
+    /**
+     * Get the type of identity that last modified the resource.
+     *
+     * @return the type of identity that last modified the resource.
+     */
+    public CreatedByType lastModifiedByType() {
+        return this.lastModifiedByType;
+    }
+
```

Comment:
@srnagar 
Let me know if you have concern with the class name `CreatedByType`. It is defined by swagger. But it might be not very appropriate (as it is returned by `createdByType()` and `lastModifiedByType()`, more like `IdentityType`?).

However `IdentityType` would cause quite some naming conflicts with other swagger types. Maybe `SystemDataIdentityType`?

Reply:
Also appreciate suggestion on method names in `SystemData`. They seems a bit off.

Reply:
IdentityType sounds like a better name than CreatedByType as it is used for lastModifiedByType too.

Reply:
SystemData seems very generic. Is this name finalized? Can we change this to AzureResourceSystemData?

Reply:
The method names look fine but just want to check, do we prefix `get` for getter methods in other arm model types?

Reply:
1. Got it.
2. It is generic. The class is intended to be returned by every ARM resources (resource group, virtual network, disk, storage account, virtual machine, etc., well, all) in future. It is already in `com.azure.core.management` namespace. On the other hand from its name, it does seems belong to something of operating system (so far, I didn't find any existing `SystemData` class that going to be confused with this). Maybe `ResourceSystemData` would be better. Let me check e.g. how python or .NET names it.
3. It is still used as POJO for ARM, so like `Resource` class, it does not have `get` prefix.